### PR TITLE
CI: Use linters to check branches before merge

### PR DESCRIPTION
**Pull request by bilbof**
This should ensure Terraform and JSON files are valid before merging.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: linting
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: macos-latest
```
Comment (issyl0) [inline .github/workflows/ci.yml:5]: You can use `ubuntu-latest` here, as the Ubuntu Actions runners now ship with Homebrew on Linux.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: terraform fmt
+        run: |
+          brew install tfenv
+          tfenv install
+          tfenv install 0.13.0-beta3
+          tfenv use 0.13.0-beta3
```
Comment (issyl0) [inline .github/workflows/ci.yml:14]:
```suggestion
```

`tfenv install` pulls from the repo's `.terraform-version` which should be in the repository root (or per-project if it needs to be different).

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: linting
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: terraform fmt
+        run: |
```
Comment (issyl0) [inline .github/workflows/ci.yml:10]:
```suggestion
        working-directory: ./terraform/
        run: |
```

The `.terraform-version` file is in `terraform`, so if we `cd` into there.

Comment (bilbof) [reply .github/workflows/ci.yml:10]: I'm not sure it's necessary if we have a `.terraform-version` at the root

**Pull request merged**